Add authorship guidelines (#1747)
celeritas-project/celeritas: doc/development/administration.rst

    and pass the CI.
#.  If releasing a backported version branch, cherry-pick this documentation
    commit into the backport branch.
#.  Use the release-notes `notebook`_ to push a draft release to GitHub.
#.  Using the GitHub interface, verify the release notes and publish it.

> .. _notebook: https://github.com/celeritas-project/release-scripts

Reviewer: Site note: the notebook appears to fail at least for me:
For https://github.com/celeritas-project/release-scripts/blob/main/release-notes.ipynb
```
TypeError                                 Traceback (most recent call last)
Cell In[45], line 1
----> 1 artifact_url, artifact_tgz = release_notes.get_or_upload_tarball(cached, release)
      2 print(f"Artifact URL: {artifact_url}")
...
TypeError: 'NoneType' object is not subscriptable
````

Author: Yep, I've been fixing it as I go along :)

Making a public API private should be treated as a deprecation.

.. [#] During initial development, deprecations will target ``v0.Y``.

.. _authorship:

Authorship and attribution
==========================

The Celeritas team have specified this policy, to give proper credit in correct
proportions to the many people who have contributed over the course of this
project. All works by the Celeritas team should be added to the Celeritas doc
repo and the `Celeritas Zotero database`_.

.. _Celeritas Zotero database: https://www.zotero.org/groups/2380941/celeritas

Code objects
------------

Each major and minor version of the code will be published in the `Celeritas
Zenodo group`_ (see :ref:`release_process`). Creator/contributor listings are
based on contributions since the *previous major release* (PRs since the "dev"
tag indicates divergence from the previous major series), ordered by level of
effort:

- Creator: contributors whose pull requests are incorporated into the release,
  ordered by number of PRs.

> - Contributor as "Editor": reviewers of PRs, ordered by number reviews

Reviewer 2: ```suggestion
- Contributor as "Editor": reviewers of PRs, ordered by number of reviews
```

- Contributor as "ProjectManager": code lead
- Contributor as "ProjectMember": core team

> - Contributor as "ProjectLeader": core advisors

Reviewer: Is the intent for names to be in general listed in 2 or 3 of those categories?  (Related question, did we already uploaded one set of those or it is still in the works)

Author: Originally I was going to deduplicate but after looking it makes more sense to have both roles: see https://zenodo.org/records/15256364

Each major version of Celeritas should be a Zenodo "concept DOI"; minor and patch
versions should be "version DOIs" (i.e., updates to the previous version).

.. _Celeritas Zenodo group: https://zenodo.org/communities/celeritas

Presentations
-------------

The title slide should credit anyone who worked directly on the presentation or
contributed new data/visualizations to the work. The presenter should be
highlighted either as first author or with emphasis. The author list should
also include "Celeritas team".

- Reusing slides from previous public presentations should contain small text

>   (I usually use thin, ultra-light italics in gray) with a citation of the

Reviewer 2: ```suggestion
  (e.g. thin, ultra-light italics in gray) with a citation of the
```

   The minimum requirements for authorship, common to all definitions, are 1)
   substantial contribution to the work and 2) accountability for the work that
   was done and its presentation in a publication. ... Acknowledgements may be
   used to denote contributions to the work that do not meet the criteria of
   authorship such as, supporting the study, general mentoring, collecting
   data, acting as study coordinator, and other related activities.

From this we have the following policies:

- Direct contributors to the article, using roles defined by `CRediT`_
  (Contributor Roles Taxonomy), get authorship credit.
- Every team member who wants to publish a paper should notify the whole team
  and give them a chance to participate directly.
- Code contributions are credited by proxy (and highlighted) through a direct
  citation to the Zenodo code object of the Celeritas version used.
- Team members whose work enables the paper are highly encouraged to describe
  the methods and implementation in the paper and thus become authors.

> - Mentoring, coordinating, funding acquisition, etc. are not sufficient in

Reviewer: This is a nit-pick.  The 'etc.' leaves ambiguous was is not sufficient.  In particular it might be clear whether reviewing or editing are sufficient or not sufficient for authorship credit.  

[Unless the first bullet was meant to say all the roles listed in `CRediT` are considered direct contributions; i.e. in which case both my example are sufficient :) ]

Reviewer: humm ... my example missed the next line which mention explicitly `(editing, revising)` ... that leaves `reviewing` (which in `CRediT`_ is bundled with writing) or is the spelling `revising` meant to be `reviewing`? (I am not sure what is the meant difference between editing and revising :) ).

Author: I updated the text; could you take another quick look? CRediT is also very broad and vague so I removed the reference.

- Direct contributors to the article get authorship credit.
- Every team member who wants to publish a paper should notify the whole team
  and give them a chance to participate directly.
- Code contributions are credited by proxy (and highlighted) through a direct
  citation to the Zenodo code object of the Celeritas version used.
- Team members whose work enables the paper are highly encouraged to describe
  the methods and implementation in the paper and thus become authors.
- Mentoring, coordinating, funding acquisition, and other supplementary roles
  are not sufficient in themselves to be marked as authors. However, these
  activities should motivate direct participation (editing, revising) as authors.

  themselves to be marked as authors. However, these activities should motivate
  direct participation (editing, revising) as authors.

.. note:: If possible, make the Celeritas release used in the article the

>    *first* reference in the docment.

Reviewer 2: ```suggestion
   *first* reference in the document.
```
